wiki/History_of_osu!/2009/en.md

```diff
@@ -35,3 +35,128 @@ osu! makes its way to the iPhone from a port to the iPhone by nuudles. The proje
+
+## February
+
+On 14 February, the fourth iteration of the osu! community meeting was held regarding how beatmaps should be moderated, a new upcoming localisation feature and questions about expanding the osu! team.[^community-meeting-4]
+
+A [kudosu](/wiki/Kudosu) system was implemented to ease beatmap [modding](/wiki/Modding) in the [beatmap ranking procedure](/wiki/Beatmap_ranking_procedure). Mappers were able to reward people with points called kudosu for giving useful and constructive criticism. 3 kudosu points could be traded with 1 kudosu star. These stars could be used to put the beatmap on a higher rank on the [Beatmap Appreciation Team (BAT)](/wiki/People/Beatmap_Appreciation_Team) modding priority list.[^kudosu-1]
+
+## March
+
+3 updates brought a lot of changes to osu! this month. The first one included filtering options for multiplayer lobbies, as well as a chat option to filter messages if offensive language appeared. If enabled, each chat message containing forbidden entries would be replaced with words defined by a certain list.[^stable-b622]
+
+In the next update being published on 21 March, [kiai time](/wiki/Gameplay/Kiai_time) was added, enabling mappers to emphasise strong parts of songs with star fountains.[^stable-b631]
+
+On 26 March, osu! received support for the TaTaCon wii Taiko drum controller and Wiimote game/menu control support (Catch The Beat only).[^stable-b639][^custom-changelog]
+
+## April
+
+This month, the registered user count exceeded the milestone of 100,000 users.[^user-count]
+
+## May
+
+A change in the kudosu system allowed users to gift kudosu stars to any beatmap, not just their own ones like previously.[^kudosu-change]
+
+Among other changes, osu! got an update this month adding widescreen support and the [Autopilot](/wiki/Gameplay/Game_modifier/Autopilot) mod. Enabling the Autopilot mod would take over the cursor movement and the player would only need to control the tapping.[^stable-b674]
+
+## June
+
+Due to a change to the [Beatmap submission system](/wiki/Beatmapping/Beatmap_submission) users were no longer able to upload any modifications to a beatmap after it was modded. There were cases of beatmaps becoming ranked without the BAT member who ranked the beatmap knowing that it contained [rule-breaking changes](/wiki/Ranking_criteria).[^bss-update]
+
+Before, osu! was only available in English. With the new localisation system coming into place, users became able to translate the game client. They could download a file containing the game's text lines, modify it to their language and ask the file to be integrated into the game client.[^client-localisation][^stable-b695]
```

Comment:
not sure if just saying "before" is enough to introduce the change [before = before what?]
```suggestion
Up to this point, osu! was only available in English. With the new localisation system coming into place, users became able to translate the game client. They could download a file containing the game's text lines, translate it into their language, and then ask for the file to be integrated into the game client.[^client-localisation][^stable-b695]
```